src/View/Helper/FormHelper.php

```diff
@@ -1676,8 +1676,8 @@ public function postLink($title, $url = null, array $options = [])
 
         $this->_lastAction($url);
         $action = $templater->formatAttributes([
-            'action' => $this->Url->build($url),
-            'escape' => false
+           'action' => $this->Url->build($url),
+           'escape' => false
```

Comment:
No, I didn't do it on my own. Seems my reformatting was broken somehow there. Sorry, will add the missing space there again.